ui/tests/integration/components/database-role-setting-form-test.js

```diff
@@ -114,20 +101,19 @@ module('Integration | Component | database-role-setting-form', function (hooks)
 
   test('it shows appropriate fields based on roleType and db plugin', async function (assert) {
```

Comment:
it seems like we could add a test for when `skip_static_role_rotation_import: true` (oof that's a long variable name) to assert the value is also set correctly when true